src/main/java/com/foober/foober/model/Review.java

```diff
@@ -0,0 +1,16 @@
+package com.foober.foober.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+public class Review {
```

Comment:
Review should link to Drive since Clients can only review drivers they've been in a drive with in the last 2 weeks IIRC